articles/api-auth/grant/hybrid.md

```diff
@@ -0,0 +1,75 @@
+---
+description: Describes how to call APIs from applications using the Hybrid Flow
+topics:
+  - authorization-code
+  - api-authorization
+  - implicit
+contentType: concept
+useCase:
+  - secure-api
+  - call-api
+---
+# Call APIs Using the Hybrid Flow
+
+The Hybrid Flow is an OpenID Connect (OIDC) flow that draws from the following:
+
+1. [Authorization Code Grant](/api-auth/grant/authorization-code)
+2. [Implicit Grant](/articles/api-auth/grant/implicit)
+
+The Hybrid Flow enables use cases where your application can immediately use an ID token to access information about the user while obtaining an authorization code that can be exchanged for an Access Token (therefore gaining access to protected resources for an extended period of time).
+
+## Background
+
+With the [Authorization Code Grant](/api-auth/grant/authorization-code), Auth0 sends you an authorization code (authorization not required), which your app then sends in to retrieve tokens (authorization required). Your application authorizes itself with a Client ID and Client Secret stored securely on your server.
+
+On the other hand, the [Implicit Grant](/articles/api-auth/grant/implicit) allows you to request Access Tokens without needing to authenticate your application. Auth0 verifies your app's identity based on the provided redirect URI. Because of this, you shouldn't utilize long-lived Access Tokens, and you cannot use Refresh Tokens.
+
+## The Hybrid Flow
+
+The Hybrid Flow allows you to take advantage of aspects of both the Authorization Code and Implicit Grants. You can request any combination of Access Tokens, ID Tokens, and authorization codes from either a server-side web application or a single page application (SPA).
```

Comment:
The hybrid flow response types ALWAYS include an authorization code, so you cannot request any combination of the three since it'll always be present. The list below is correct tho.